ocaml/xcp-rrdd/bin/rrdp-scripts/rrdd-plugins-init

```diff
@@ -37,9 +34,8 @@ start() {
 		
 		if [ -e ${SUBSYS_FILE} ]; then
```

Comment:
Would be good to completely drop this script and use a systemd service file instead. Is that possible?
We don't need to keep all these lock/pid files around, systemd can take care of ensuring only 1 instance is running.

Reply:
this file is not in use, just dropped